scripts/quests/jeuno/Wings_of_Gold.lua

```diff
@@ -0,0 +1,99 @@
+-----------------------------------
+-- Wings of Gold
+-----------------------------------
+-- Log ID: 3, Quest ID: 60
+-- Brutus : !pos -55 8 95 244
+-----------------------------------
+local quest = Quest:new(xi.questLog.JEUNO, xi.quest.id.jeuno.WINGS_OF_GOLD)
```

Comment:
Just one more request: Can you please annotate above this line with `---@type TQuest`?

Reply:
Sure thing! 👍🏻